### PR TITLE
Blast from the past rust setup

### PR DESCRIPTION
**Pull request by ericwburden**
Rust challenge setup for Blast from the Past

### CODE REVIEW

**Review by xanderyzwich — APPROVED** [review]

File: challenges/2022-10-11-blast-from-the-past/setups/rust/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[package]
+name = "rust"
+version = "0.1.0"
+edition = "2021"
```
Comment (parhaml) [inline challenges/2022-10-11-blast-from-the-past/setups/rust/Cargo.toml:4]: Ooooohhh! Not nightly? 😉

File: challenges/2022-10-11-blast-from-the-past/setups/rust/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```
Comment (parhaml) [inline challenges/2022-10-11-blast-from-the-past/setups/rust/Cargo.toml:8]: Is this required even with nothing defined?

File: challenges/2022-10-11-blast-from-the-past/setups/rust/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        let expected = [Some(-7), Some(5), Some(11)];
+        assert_eq!(result, expected);
+    }
+
+    #[test]
+    fn test_case_three() {
+        let input = [1];
+        let result = find_three_largest_numbers(&input);
+        let expected = [Some(1), None, None];
+        assert_eq!(result, expected);
```
Comment (parhaml) [inline challenges/2022-10-11-blast-from-the-past/setups/rust/src/lib.rs:48]: Same question as with your Julia. Is it possible to define `null` as a returnable value?

**Review by parhaml — APPROVED** [review]
Just nitpicky questions!

File: challenges/2022-10-11-blast-from-the-past/setups/rust/src/lib.rs (hunk 3 above)
Comment (drkennetz) [reply challenges/2022-10-11-blast-from-the-past/setups/rust/src/lib.rs:48]: same question here from me before merging. Are there no "zero" values for these?

File: challenges/2022-10-11-blast-from-the-past/setups/rust/Cargo.toml (hunk 1 above)
Comment (ericwburden) [reply challenges/2022-10-11-blast-from-the-past/setups/rust/Cargo.toml:4]: No need to live on the "edge" for these, lol

File: challenges/2022-10-11-blast-from-the-past/setups/rust/Cargo.toml (hunk 2 above)
Comment (ericwburden) [reply challenges/2022-10-11-blast-from-the-past/setups/rust/Cargo.toml:8]: Nah, it's just auto-generated from cargo.

File: challenges/2022-10-11-blast-from-the-past/setups/rust/src/lib.rs (hunk 3 above)
Comment (ericwburden) [reply challenges/2022-10-11-blast-from-the-past/setups/rust/src/lib.rs:48]: Yep. I commented on the other, but you're right. I'll fix it.

**Review by drkennetz — APPROVED** [review]

**Review by mrumiker — APPROVED** [review]

**Pull request merged**